exasol_integration_test_docker_environment/cli/options/test_environment_options.py

```diff
@@ -19,6 +19,9 @@
     click.option('--docker-db-image-name', type=str, default="""exasol/docker-db""",
                  show_default=True,
                  help="""Docker DB Image Name against which the tests should run."""),
+    click.option('--docker-access-method', type=click.Choice(['DOCKER_EXEC', 'SSH']),
```

Comment:
```suggestion
    click.option('--docker-access-method', type=click.Choice([e.name for e in DockerAccessMethod]),
```

Reply:
Thanks to @Nicoretti -- I like this idea and will apply it in the next update of this PR.

I had discussed that with @tkilias before and was not sure if we want to add the required import:
```python
from exasol_integration_test_docker_environment \
    .lib.test_environment.parameter \
    .docker_db_test_environment_parameter \
    import DockerAccessMethod
```